Woof/WoofTests/Helper/AppStyleTests/AppStyle+UIElementConstantTests.swift

```diff
@@ -6,10 +6,12 @@ final class UIElementConstantTests: XCTestCase {
         let cornerRadius = 10.0
         let minPadding = 4.0
         let avatarSideSize = 80.0
+        let vStackSpacing = 8.0
```

Comment:
nit:

Am I right I have to use that spacing for an every `VStack`?

Reply:
No, not for every.
Fixed the description of the constant.

Reply:
I'm sorry, could you please explain what was fixed?

Reply:
I mean, that I rewrite the doc comment to the constant. 
Now I also change the constant's name.